### DISCUSSION TITLE
[Lesson 2][Concept 11]Democracy & Healthy Public Conversation - Resources

### DISCUSSION THREAD

**Discussion by cescapa** [OpenMined/courses#130] [Course: Our Privacy Opportunity] [answered]
At 8'38" Andrew mentions "gov zero" and "v taiwan". It's not clear what they are. Could I suggest including further information in a resources section, below the quiz?

**Comment by em-blue** [comment, marked as answer]
Great point, we're getting around to adding all the extra resources as soon as we can :-)
Here are some for that lesson & movement:

The g0v movement: https://en.wikipedia.org/wiki/G0v
vTaiwan: https://info.vtaiwan.tw/

[The simple but ingenious system Taiwan uses to crowdsource its laws ](https://www.technologyreview.com/2018/08/21/240284/the-simple-but-ingenious-system-taiwan-uses-to-crowdsource-its-laws/)
[Airbox: A Community-Driven Effort to Improve Our Air](https://www.taiwan-panorama.com.tw/en/Articles/Details?Guid=e34da4f0-b24c-4d88-9f63-e8ab3eee5aaa&CatId=9)

If you google "g0v" "vTaiwan" and "Audrey Tang" you'll find a lot more - it's been covered here and there in the news. Glad you're interested in the movement! :)

**Reply by em-blue** [reply]
Just added some links to all videos mentioning g0v and vTaiwan! Thanks for the suggestion! Let us know if you have any others

**Comment by nahuakang** [comment]
![resources](https://user-images.githubusercontent.com/18533347/103583751-9981e600-4ee0-11eb-97a4-7b5f8d745773.png)
Yeah, some lessons already have links provided now :+1:

**Reply by em-blue** [reply]
@nahuakang by the way, if you have suggestions for certain resources to add let me know and I'll add them in :-)
